infra-as-code/bicep/modules/vnetPeering/vnetPeering.bicep

```diff
@@ -1,25 +1,27 @@
-@description('Virtual Network ID of Virtual Network destination. No default')
+@description('Virtual Network ID of Virtual Network destination.')
 param parDestinationVirtualNetworkId string
 
-@description('Name of source Virtual Network we are peering. No default')
+@description('Name of source Virtual Network we are peering.')
 param parSourceVirtualNetworkName string
 
-@description('Name of destination virtual network we are peering. No default')
+@description('Name of destination virtual network we are peering.')
+@minLength(2)
+@maxLength(64)
```

Comment:
Breaking, if exceeded maxLength

Reply:
As per RSGs wouldn't be breaking as platform limit so no one would have been able to deploy it any longer anyway